s64da_benchmark_toolkit/prepare.py

```diff
@@ -176,18 +176,26 @@ def _load_pre_schema(self, conn):
                 conn.cursor.execute(pre_schema_file.read())
 
     def _load_license(self, conn):
-        license_path = '/s64da.license'
         try:
+            conn.cursor.execute(f'select swarm64da.show_license()')
+            license_path = self.args.s64da_license_path
             conn.cursor.execute(f'select swarm64da.load_license(\'{license_path}\')')
```

Comment:
the error message generated if that failes now is:
Traceback (most recent call last):
  File "prepare_benchmark", line 88, in <module>
    prepare.PrepareBenchmark(args, benchmark).run()
  File "/home/nethomes/juri.schmidt/s64da-benchmark-toolkit2/s64da_benchmark_toolkit/prepare.py", line 148, in run
    self.prepare_db()
  File "/home/nethomes/juri.schmidt/s64da-benchmark-toolkit2/s64da_benchmark_toolkit/prepare.py", line 229, in prepare_db
    self._load_license(conn)
  File "/home/nethomes/juri.schmidt/s64da-benchmark-toolkit2/s64da_benchmark_toolkit/prepare.py", line 180, in _load_license
    conn.cursor.execute(f'select swarm64da.show_license()')
psycopg2.errors.InternalError_: No valid license found.
CONTEXT:  Swarm64 DA


This seems not intuitive to me. I would then rather check if the file is available and fail with a meaningful error.
Also since a license file is required it should also be documented in the readme.